conda-forge.yml

```diff
@@ -1,3 +1,4 @@
+azure: {store_build_artifacts: true}
```

Comment:
I assume this is a leftover from testing?

Reply:
Leftover in the sense that it was added while I was testing, but I find it generally useful to have in place all the time so you don't have to stick it in and re-run the CI to analyze the output.